package.nix

```diff
@@ -0,0 +1,15 @@
+{ pkgs ? import <nixpkgs> {}, displayrUtils }:
+
+pkgs.rPackages.buildRPackage {
+  name = "verbs";
+  version = displayrUtils.extractRVersion (builtins.readFile ./DESCRIPTION); 
+  src = ./.;
+  description = "A collection of operations/actions/"verbs" providing an intuitive interface for many common operations (e.g. Sum, First, Average, Maximum, ...) on various inputs/data structures. Created to simplify common user actions in Displayr with a wide variety of inputs.";
```

Comment:
Did you build this? I'd expect it to fail because we terminate the string early, and that syntax shouldn't be valid. I suppose it is possible nothing in the build depends on the description so it is never evaluated, but even the syntax should cause the evaluation to fail. 

Whether it builds or not we should fix the description here.